store/postgres/src/connection_pool.rs

```diff
@@ -773,7 +773,7 @@ impl PoolInner {
             // closure failed.
             let conn = pool
                 .get()
-                .map_err(|e| CancelableError::Error(StoreError::Unknown(e.into())))?;
```

Comment:
Do we still use the `Unknown` variant for other `StoreError` scenarios?

Reply:
Yeah, there are a ton of uses of it all over the code.